src/components/Jobs/DetailJobPage.jsx

```diff
@@ -49,18 +51,26 @@ class DetailJobPage extends Component {
         }
       })
       .then(res => {
-        this.setState({ 
-          jobDetail:{
+        this.setState({
```

Comment:
JobDetails = Object.assign({}, res.data.response)

jobDetails.start_time = moment((new Date(jobDetails.start_time)).lang("vi")

this.setState({jobDetail: JobDetails})